Dockerfile

```diff
@@ -1,11 +1,33 @@
-FROM python:2.7 
+# getting the base Image
+FROM python:3.6
+
 MAINTAINER Ana Balica <ana.balica@gmail.com>
```

Comment:
It would be better we have an official Email ID here

Reply:
Yes so I can remove my email.